src/infrastructure/network/fakeRequest.ts

```diff
@@ -0,0 +1,11 @@
+/**
+ * fakeRequest useful for debug: it prints and returns the same object input
+ */
+import { RequestT } from './types';
+
+const request: RequestT = async ({ basePath, authorization, method, path, body }) => {
+  console.log({ basePath, authorization, method, path, body });
+  return JSON.parse(JSON.stringify({ basePath, authorization, method, path, body }));
```

Comment:
instead to serialize and unserialize object you can return

`return { basePath, authorization, method, path, body } as any`

doing that the request is compliant with the interface

Reply:
Ok, many thanks! I didn't know it (this is my first attempt on TS).
BTW I've just deleted this class because it is useless now